tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/BootstrapTasks/GetBuildTools.cs

```diff
@@ -25,7 +25,7 @@ public class GetBuildTools : Task
         const string LKG_AKAMS_URI = @"http://aka.ms/AzNetSDKBuildTools";
         const string LATEST_AKAMS_URI = @"http://aka.ms/LatestBuildTools";
 
-        const string COPY_TO_RELATIVEPATH = @"tools\SdkBuildTools\";
+        const string COPY_TO_RELATIVEPATH = @"tools\BuildToolsForSdk\";
```

Comment:
@iansul are you getting any errors that prompted you to do this?
Because now we do not rely on these branch names, we get the branch name by making a HEAD request to an external URI that gives us the current branch name.
Those URI are specified in line 25 and 26.

So would like to understand your scenario that needs this change, has anything not worked for you?